index.html

```diff
@@ -0,0 +1,243 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8" />
+
+	<title>ToDo List</title>
+	<meta name="description" content="ToDo application" />
+	<meta name="author" content="Bartlomiej Zdybel" />
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+	<link rel="stylesheet" type="text/css" href="./index.css" />
+
+	<link rel="icon" href="icon_code.jpg" />
+</head>
+
+<body>
+	<div class="container">
+		<div class="task-list__container">
+			<form class="task-form">
+				<h1 class="task-form__title">task list</h1>
+
+				<div class="task-form__input-wrapper">
+					<label class="task-form__label" for="task">What must you do?</label>
+					<input id="task" class="task-form__input task-form__input--task-content"
+						placeholder="eg. Go to the cinema" />
```

Comment:
Is this code formatted with Prettier? I've added a comment about this (already resolved), but I'm not sure

Reply:
Yes, prettier is working.